admin/app/views/workarea/admin/catalog_variants/index.html.haml

```diff
@@ -49,6 +50,11 @@
               %td.align-right
                 = link_to pricing_sku_path(variant.sku) do
                   = number_to_currency variant.pricing.sell_price
+              %td.align-right
+                = link_to shipping_sku_path(variant.sku), class: 'index__table-multi-line-link' do
+                  = variant.weight
+                  %br
+                  = variant.dimensions
```

Comment:
Why does this need to be two lines? Not sure we need to present both pieces of data on this table, and if so maybe it should just be two columns `Weight` and `Dimensions` with each value linking to shipping sku.